### PR TITLE
filter by grant tags

### PR DESCRIPTION
**Pull request by bnchdrff**

### CODE REVIEW

File: src/app/views/grants/container.js [hunk 1]
```diff
@@ -37,26 +39,34 @@ var GrantContainerView = Backbone.View.extend({
...
   },
 
   setUpFilter: function() {
     this.$el.find('.org-search input').on('keyup', _.throttle(this.handleFilter.bind(this), 0));
+    this.$el.find('.org-search select.filterbytag').on('change', this.handleFilter.bind(this));
   },
 
-  afterRender: function() {
-    this.setUpFilter();
+  setGrantTagsAndRenderFilters: function(grant_tags) {
```
Comment (hampelm) [inline src/app/views/grants/container.js]: This is quite neatly done. How much slower would it be to pull the `grant_tags` using a method on the collection? I'm a little leery of the parent getting the data from the child, feels out of the typical backbone pattern.

File: src/app/views/grants/list.js [hunk 2]
```diff
@@ -50,15 +52,25 @@ var GrantListView = Backbone.View.extend({
...
       var sum = _.reduce(grants, function(memo, grant) {
-        // along the way, build our yearly sums!
+        // along the way, build our yearly sums & grant tags.
         var this_year = grant.field_year.value.slice(0,4);
         if (yearly_sums[this_year] > 0) {
           yearly_sums[this_year] += grant.field_funded_amount;
         } else {
           yearly_sums[this_year] = grant.field_funded_amount;
         }
+        if (grant.field_grant_tags && grant.field_grant_tags.hasOwnProperty('und')) {
```
Comment (hampelm) [inline src/app/views/grants/list.js]: I theory all the data prep that happens in `GrantListView` should be in the collection but I don't think it's worth moving it now. Def. out of scope for this work.

**Comment by hampelm** [comment]
Overall looks good to me. Good to merge unless you want to make the change noted in the review.

File: src/app/views/grants/container.js (hunk 1 above)
Comment (bnchdrff) [inline src/app/views/grants/container.js]: yeah that totally makes sense. i just following suit with that existing code to do totals!

i'll take a stab at refactoring real quick.

**Pull request merged**